classes/rental.rb

```diff
@@ -0,0 +1,19 @@
+class Rental
+  attr_accessor :date
+
```

Comment:
`:book` and `:person` should be added to the `attr_accessor` here to allow access to viewing and adding books and person 👍🏽